### ISSUE TITLE
0.55.2 no longer reporting the hostname (only ip) in sys.nodes

### ISSUE THREAD

**Issue by jodok** [crate/crate#3798]
crate 0.54.9:
```
➜  ~ crash
cr> select hostname from sys.nodes;
+-------------+
| hostname    |
+-------------+
| muire.local |
+-------------+
SELECT 1 row in set (0.011 sec)
cr>
```

crate 0.52.2:
```
➜  ~ crash
cr> select hostname from sys.nodes;
+----------------+
| hostname       |
+----------------+
| 192.168.39.165 |
+----------------+
SELECT 1 row in set (0.525 sec)
cr>
```

**Comment by kovrus** [comment]
@jodok it has been fixed on ``master`` and backported to ``0.55``. The fix will be released with ``0.55.3``.

**Comment by kovrus** [comment]
#3823
